src/main/java/se/kth/sda/skeleton/posts/PostController.java

```diff
@@ -11,12 +11,10 @@
 @RequestMapping("/posts")
 @RestController
 public class PostController {
-    PostRepository postRepository;
     PostService postService;
 
```

Comment:
What is done here ?
If we did that it means that our class `postRepository `is useless, no? but we need it
Could you explain

Reply:
Yes, we don't use it in PostController, but in PostService...So, as we extracted to PostService, it's redundant to have it here anymore

Reply:
alright, good for me then